content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -1040,6 +1040,12 @@ N/A
 @id
 ```
 
+As of 2.29.0, if the `namedConstraints` preview flag is enabled the signature changes:
```

Comment:
```suggestion
As of 2.29.0, if the `namedConstraints` preview feature flag is enabled the signature changes to:
```

We should have the same sentence everywhere, it is currently slightly different across the page.